packages/editor/src/client/panel/imageEditPanel.tsx

```diff
@@ -193,6 +195,62 @@ export function ImagedEditPanel({id, file, onClose, onSave}: ImageEditPanelProps
     }
   }
 
+  /**
+   * Resizes an image on client side, if larger than 10 MB, which is the upload max of the api
+   * @param file
+   */
+  async function resizeImage(file: File): Promise<File> {
+    // only resize image if larger than 10 MB
+    if (!willImageResize(file)) {
+      return file // do not resize
+    }
+    const originalFileName: string = file.name
+    if (!originalFileName) {
+      throw new Error('No file name provided!')
+    }
+    const options = {
+      targetSize: 10, // the max size in MB, defaults to 2MB
```

Comment:
I would compress it to the the size that we defined in the environment variable. If i set the minimum to 50mb and then upload an image of 51mb i don't want it resized to 10mb.

Reply:
yeah. of course.